packages/client/modules/meeting/components/TemplateScaleValueItem.tsx

```diff
@@ -66,11 +64,12 @@ const ScaleAndDescription = styled('div')({
 
 const TemplateScaleValueItem = (props: Props) => {
   const {dragProvided, isDragging, isOwner, scale, scaleValue} = props
+  const {id: scaleId} = scale
+  const {label, color} = scaleValue
   const [isHover, setIsHover] = useState(false)
   const [isEditingScaleValueLabel] = useState(false)
   const {submitting, submitMutation, onError, onCompleted} = useMutationProps()
   const atmosphere = useAtmosphere()
-  const canRemove = !scaleValue.isSpecial
```

Comment:
i think this was an artifact of some old logic. since ? and Pass are always there, we can allow folks to go down to 0 user-defined values